src/views/ScorePage/ScorePage.js

```diff
@@ -0,0 +1,86 @@
+import './ScorePage.css';
+import { scoreLocalStorage, saveHighScore } from '../../score/user-score';
+import Button from '../../components/Button';
+import '../../components/Button.css';
+
+export function renderScorePage() {
+  const highScores = JSON.parse(localStorage.getItem('highScores')) || [];
+  const recentUserScore = 12;
+  const questionsNumber = 12;
+
+  const container = document.createElement('div');
+  container.setAttribute('id', 'scorePage');
+  container.append(
+    renderGraphics(),
+    RenderScore(recentUserScore, questionsNumber),
+    renderNickForm(),
+    renderMenuButton(),
+    renderPlayAgainButton(),
+  );
+  document.querySelector('#app').append(container);
+}
+
+function RenderScore(score, questionsNumber) {
+  const container = document.createElement('div');
+  const congratsText = document.createElement('h2');
+  const scoreText = document.createElement('h2');
+  const finalScore = document.createElement('h2');
+  finalScore.setAttribute('id', 'finalScore');
+  congratsText.innerText = 'CONGRATS!';
+  scoreText.innerText = 'YOUR SCORE IS: ';
+  container.append(congratsText, scoreText, finalScore);
+  return container;
+}
+
+function renderGraphics() {
+  const img = document.createElement('img');
+  img.setAttribute('src', '../../public/cat.png');
```

Comment:
Double-check if this path is correct for a build (npm run build && npm run start - to check the preview). It will work fine with ./cat.png